filter/options.go

```diff
@@ -10,7 +10,7 @@ import (
 type Options struct {
 	Options []string `arg:"" optional:"" help:"Options to filter."`
 
-	Indicator             string        `help:"Character for selection" default:"•" env:"GUM_FILTER_INDICATOR"`
+	Indicator             string        `help:"Character for selection" default:"• " env:"GUM_FILTER_INDICATOR"`
```

Comment:
Is this espace addition made on purpose?

Reply:
yes

this is still very wip though